addons/stock/tests/test_quant.py

```diff
@@ -28,6 +28,13 @@ def setUpClass(cls):
             'notification_type': 'inbox',
             'groups_id': [(6, 0, [cls.env.ref('stock.group_stock_user').id])]
         })
+        cls.user_stock_manager = Users.create({
```

Comment:
test_quant.py should only do sudo() and never use the inventory mode
these are some low levels tests

Reply:
actually nothing should be touched at all in this file in my opnion